src/main/java/com/iheartradio/m3u8/data/ByteRange.java

```diff
@@ -0,0 +1,56 @@
+package com.iheartradio.m3u8.data;
+
+import java.util.Objects;
+
+public class ByteRange {
+    private final long mSubRangeLength;
+    private final Long mOffset;
```

Comment:
Please use a primitive, see `IFrameStreamInfo.getAverageBandwidth` as an example.

Reply:
Actually, there is no `null` case for the offset. I'll refer to my comment here:

https://github.com/iheartradio/open-m3u8/pull/47#discussion_r130231079